src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,128 @@
+package duke;
+
+import duke.exception.InvalidCommandException;
+import duke.exception.InvalidDescriptionException;
+import duke.task.Task;
+
+import java.util.Scanner;
+
+public class Duke {
+    public static final String HALF_TAB = "  ";
+    public static final String BREAK = System.lineSeparator();
+    public static final String TAB = "    ";
+    public static final String BORDER = "    ____________________________________________________________";
```

Comment:
Besides set the border as a constant string, it is also okay to create a printline() method, it might also be easier for you to add \n at the end of constant string if java allows